packages/big-design/src/components/Pagination/styled.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+import { StyleableButton } from '../Button/private';
+
+export const StyledButton = styled(StyleableButton)`
+  color: ${({ theme }) => theme.colors.secondary70};
+`;
+
+// in case we need to add separate styles to the icon buttons or to the dropdown trigger
+export const StyledIconButton = styled(StyleableButton)`
```

Comment:
We probably don't want another button unless we absolutely need it. You could get rid of this and just focus on styling one button above.